dashboard/config/routes.rb

```diff
@@ -271,6 +271,12 @@
     end
   end
 
+  # Redirects from old /stage url to new /lesson url
+  get '/s/:script_name/stage/:position(*all)', to: redirect(path: '/s/%{script_name}/lesson/%{position}%{all}')
+
+  # Redirects from old /lockable url to new /survey url
+  get '/s/:script_name/lockable/:position(*all)', to: redirect(path: '/s/%{script_name}/survey/%{position}%{all}')
+
```

Comment:
These lines are the core of this PR. They redirect old URLs to the new URL scheme.